Microsoft.Azure.CosmosRepository/src/FullItem.cs

```diff
@@ -31,15 +32,39 @@ namespace Microsoft.Azure.CosmosRepository
     /// ]]>
     /// </code>
     /// </example>
-    public abstract class FullItem : Item, IItemWithEtag
+    public abstract class FullItem : Item, IItemWithEtag, IItemWithTimeToLive, IItemWithTimeStamps
     {
-        /// <summary>
-        /// Etag for the item which was set by Cosmos the last time the item was updated. This string is used for the relevant operations when specified.
-        /// </summary>
-        /// <remarks>
-        /// Will only be used if the verifyEtag flag is specified on the relevant methods.
-        /// </remarks>
+        /// <inheritdoc />
         [JsonProperty("_etag")]
         public string Etag { get; private set; }
+
+        /// <inheritdoc />
+        public TimeSpan? TimeToLive
+        {
+            get => _timeToLive.HasValue ? TimeSpan.FromSeconds(_timeToLive.Value) : null;
+            set
+            {
+                if (value?.TotalSeconds != null) _timeToLive = (int)value?.TotalSeconds;
+            }
+        }
+
+        [JsonProperty("ttl", NullValueHandling = NullValueHandling.Ignore)]
+        private int? _timeToLive;
+
+        /// <inheritdoc />
+        [JsonIgnore]
+        public DateTime LastUpdatedTimeUtc => new DateTime(1970, 1, 1, 0, 0, 0).AddSeconds(LastUpdatedTimeRaw);
```

Comment:
We could extract this to a static readonly field, saves it been created new everytime, what do you think?